frontend/src/service/index.js

```diff
@@ -85,6 +99,15 @@ function handleErrors(response) {
         throw Error(response.statusText);
     }
 
+    if (response.status === 202) {
+        return response.json().then(({ warning }) => {
+            const error = new Error(warning);
```

Comment:
I decided against subclassing Error here because it's quite faffy in JS and it didn't feel worth it just for this one case. No strong opinion though

Reply:
Agreed